WIP-Structure-Built

PengyuW007 opened the pull request.

PengyuW007 committed:
Update main.py

PengyuW007 committed:
Update main.py

PengyuW007 committed:
CHENZHIYI

PengyuW007 committed:
Update main.py

PengyuW007 committed:
Initialize data from excel

PengyuW007 committed:
Update main.py

PengyuW007 committed:
Update main.py

PengyuW007 committed:
Update main.py

PengyuW007 committed:
Update main.py

CZUBRYTT commented:
The main code looks good so far.

CZUBRYTT reviewed (approved):
The main code looks good so far.

mayue1998 committed:
Update README.md

mayue1998 committed:
test

just for testing

mayue1998 committed:
debugged

debugged

mayue1998 committed:
still trying

1

PengyuW007 committed:
Update main.py

Lucasbeast committed:
Update main.py

PengyuW007 committed:
Create COVID19-eng.csv

PengyuW007 committed:
Merge branch 'WIP-Structure-Built' of https://github.com/Skadoosh777/COMP4710_Group14_Analyzer into WIP-Structure-Built

PengyuW007 committed:
Update main.py

PengyuW007 committed:
Update main.py

PengyuW007 committed:
Update main.py

mayue1998 committed:
data reduction

Filter unusable data by determine if these data are not stated

mayue1998 committed:
update the summary

update the summary of how many people are filtered by different conditions

PengyuW007 reviewed (commented):
Looks so good Martin.

Zhiyi233 reviewed (approved):
The main.py looks pretty nice. Good job Martin!

Lucasbeast committed:
this is nice

Lucasbeast committed:
this is nice

Lucasbeast committed:
this is nice

PengyuW007 committed:
fixed typo

PengyuW007 committed:
fixed typo

PengyuW007 committed:
xml added

Zhiyi233 committed:
sd number

Lucasbeast committed:
this is nice

Lucasbeast committed:
this is nice

PengyuW007 committed:
Create .gitignore

PengyuW007 committed:
Merge branch 'WIP-Structure-Built' of https://github.com/Skadoosh777/COMP4710_Group14_Analyzer into WIP-Structure-Built

Zhiyi233 committed:
Merge branch 'WIP-Structure-Built' of https://github.com/Skadoosh777/COMP4710_Group14_Analyzer into WIP-Structure-Built

mayue1998 committed:
debug

debug

mayue1998 committed:
first version of data visualization

first version of data visualization for death rate in different regions only  right now, will add more later

PengyuW007 committed:
comments install matplotlib

PengyuW007 reviewed (commented):
Please add all the packages into README, or if you don't know how to modify README file, please tell me then I'll add them later.

PengyuW007 commented on README.md:
```diff
@@ -6,8 +6,8 @@
 1. Pengyu Wang, 7863401
 2. Thomas Czubryt, 
 3. Hantong Li(Lucas),
```
Please add your student ID @CZUBRYTT and @Lucasbeast

PengyuW007 reviewed (commented):
Can you make some comments, which follows programming standard, to make code more readable? If markers doesn't know what are the meanings of anywhere, then he can check comments, that would be nicer.

mayue1998 committed:
collection For UVIPER

Description needed, will add comments tomorrow

CZUBRYTT committed:
Added student ID

CZUBRYTT commented:
Great work so far!
Based on the code, the PER_ASM column in probability.csv refers to the percentage of people who are SYMPTOMATIC, not asymptomatic, since the code uses `data1 = testFile.query('COV_ASM == 2')`. If you want the percentage of people who are ASYMPTOMATIC, you need to do `data1 = testFile.query('COV_ASM == 1')`. Also, the PER_DTH column refers to the percentage of people who have NOT died, since the code uses `data2 = testFile.query('COV_DTH == 2')`. If you want the percentage of people who HAVE died, you need to do `data2 = testFile.query('COV_DTH == 1')`. Also, the PER_HSP column refers to the percentage of people who have been hospitalized AND have been in intensive care. I'm not sure whether or not whether or not you also want to include people who have been hospitalized but are not in intensive care, but if you want to, you'll need to do something like `data0 = testFile.query('COV_HSP <= 2')`.

PengyuW007 committed:
Lucas Viper

PengyuW007 committed:
Lucas version

PengyuW007 committed:
fixed typos

PengyuW007 committed:
Update README.md

CZUBRYTT committed:
Fixed some text

Zhiyi233 commented:
> 

Thomas, Lucas has relied your question on the Slack.

Zhiyi233 commented:
> Great work so far! Based on the code, the PER_ASM column in probability.csv refers to the percentage of people who are SYMPTOMATIC, not asymptomatic, since the code uses `data1 = testFile.query('COV_ASM == 2')`. If you want the percentage of people who are ASYMPTOMATIC, you need to do `data1 = testFile.query('COV_ASM == 1')`. Also, the PER_DTH column refers to the percentage of people who have NOT died, since the code uses `data2 = testFile.query('COV_DTH == 2')`. If you want the percentage of people who HAVE died, you need to do `data2 = testFile.query('COV_DTH == 1')`. Also, the PER_HSP column refers to the percentage of people who have been hospitalized AND have been in intensive care. I'm not sure whether or not whether or not you also want to include people who have been hospitalized but are not in intensive care, but if you want to, you'll need to do something like `data0 = testFile.query('COV_HSP <= 2')`.

Lucas has replied your question on the Slack.

Lucasbeast committed:
this is nice

Lucasbeast committed:
Update main.py

Lucasbeast committed:
this is nice

Lucasbeast committed:
this is nice

CZUBRYTT committed:
Fixed y-axis labels
